classes/class-wc-connect-shipping-label.php

```diff
@@ -132,6 +132,42 @@ protected function get_states_map() {
 			return $result;
 		}
 
+		public function should_show_meta_box() {
+			global $post, $theorder;
+
+			if ( ! is_object( $post ) ) {
+				return FALSE;
+			}
+
+			if ( ! is_object( $theorder ) ) {
+				$theorder = wc_get_order( $post->ID );
+			}
+
+			if ( ! $theorder ) {
+				return FALSE;
+			}
```

Comment:
Can we just use `wc_get_order()`, without any arguments, instead of checking global variables?